ob-julia-vterm.el

```diff
@@ -98,10 +96,7 @@ using Logging: Logging; open(\"%s\", \"w\") do io
         Base.invokelatest(print, io, result)
         result
     catch e
-        Logging.with_logger(logger) do
-            @error e
-        end
-        e
+        showerror(logger.stream, e)
```

Comment:
I would like all results and errors to be echoed in the REPL. Can I ask the following change?

```suggestion
        msg = sprint(showerror, e)
        println(logger.stream, msg)
        println(msg)
```